dags/web_preaggregated_daily.py

```diff
@@ -188,29 +188,44 @@ def partition_web_analytics_data_by_team(
     successfully_team_ids = []
     failed_team_ids = []
 
-    for team_id in team_ids:
-        team_s3_path = f"{source_s3_path.replace('.native', '')}/{team_id}/data.native"
+    session = chdb.session.Session()
+    try:
+        temp_db = f"temp_analytics_{context.run_id.replace('-', '_')}"
+        session.query(f"CREATE DATABASE IF NOT EXISTS {temp_db} ENGINE = Atomic")
+
+        temp_table = f"{temp_db}.source_data"
+
+        session.query(f"""
+            CREATE TABLE {temp_table} ENGINE = Memory AS
```

Comment:
This is great! I am not sure why I assumed the AggregateFunction field types would not work; maybe I've read somewhere, but never really tested it.